splunk/scripts/log_indexing.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+logs_path="$SPLUNK_HOME/test_logs"
+endpoint="https://localhost:8089/services/data/inputs/oneshot"
+SPLUNK_PASSWORD="Password"
+
+echo "Indexing logs..."
+for file in "$logs_path"/*; do
+  echo "Indexing $file"
+  curl  --insecure --user admin:"$SPLUNK_PASSWORD" "$endpoint" --data name="$file" --data index="test_index"
+done
+
+timeout_start=$(date +%s)
+while true; do
+    if ! curl --insecure --user admin:"$SPLUNK_PASSWORD" "$endpoint" --get --data output_mode=json | jq -e '.paging.total == 0'; then
```

Comment:
newlines are good for your soul... For example the `;` between `if` and `then` could be a newline...

Reply:
fixed